README.md

```diff
@@ -438,10 +445,11 @@ The following plots provide a global view of the trained model along with its pr
 
 |Plot|Description|
 |----|-----------|
-|Data Exploration| An overview of the dataset along with prediction values.|
-|Global Importance|Shows the top K (configurable K) important features globally. This chart is useful for understanding the global behavior of the underlying model.|
-|Explanation Exploration|Demonstrates how a feature is responsible for making a change in model’s prediction values (or probability of prediction values). |
-|Summary| Uses a signed local feature importance values across all data points to show the distribution of the impact each feature has on the prediction value.|
+|Data Exploration| Displays an overview of the dataset along with prediction values.|
+|Global Importance|Aggregates feature importance values of individual datapoints to show the model's overall top K (configurable K) important features. Helps understanding of underlying model's overall behavior.|
+|Explanation Exploration|Demonstrates how a feature affects a change in model’s prediction values, or the probability of prediction values. Shows impact of feature interactions.|
+|Summary Importance|Uses indiviual feature importance values across all data points to show the distribution of each feature's impact on the prediction value. Using this diagram, you investigate in what direction the feature values affects the prediction values.
```

Comment:
in what direction the feature values affects the prediction values. ->
in what direction the feature values **affect** the prediction values.